4/Dockerfile

```diff
@@ -5,7 +5,7 @@ ARG NODEJS_HOME=/opt/nodejs
 ARG UID=1000
 ARG GID=1000
 ENV SONAR_SCANNER_HOME=${SONAR_SCANNER_HOME} \
-    SONAR_SCANNER_VERSION=4.2.0.1873 \
```

Comment:
not important - but maybe upgrade nodejs to recent version as well?

Reply:
as i'm not sure of the impact, i prefer not right here